runtime/src/bank.rs

```diff
@@ -19479,6 +19479,7 @@ pub(crate) mod tests {
         new_balance: u64,
         mock_program_id: Pubkey,
         recent_blockhash: Hash,
+        tx_data_size_limit: u32,
```

Comment:
Initially added the exact tx data size limit as a way to demonstrate the usage of `set_accounts_data_size_limit()`. But agree this may pollute this test. Replaced it with `u32::MAX`